src/tools/illink/.editorconfig

```diff
@@ -1,6 +1,3 @@
-# top-most EditorConfig file
-root = true
-
```

Comment:
Do we want to consolidate the code styling in src/tools/illink with the rest of the repo? If yes, can you please open a tracking issue to remove this .editorconfig later?

Reply:
Once the linker repo is archived and no cherry picking is being made the editorconfig will be customized to only contain a small subset of overrides from runtime global config https://github.com/dotnet/runtime/issues/78050